packages/angular/cli/lib/config/schema.json

```diff
@@ -398,6 +401,33 @@
         "^[a-z]{1,3}-.*": {}
       },
       "definitions": {
+        "i18n": {
+          "description": "Project i18n options",
+          "type": "object",
+          "properties": {
+            "missingTranslation": {
+              "type": "string",
+              "description": "How to handle missing translations for i18n.",
+              "enum": ["warning", "error"]
+            },
+            "sourceLocale": {
+              "type": "string",
+              "description": "The locale id used in the source code.",
+              "default": "en-US"
+            },
+            "locales": {
+              "type": "object",
+              "additionalProperties": false,
+              "patternProperties": {
+                "^[a-z]{2}(-[a-zA-Z]{2,})?$": {
+                  "type": "string",
```

Comment:
Following a chat with @petebacondarwin, this might become an array to support multiple input files for the same locale

Reply:
We can expand this as needed.  The string form will generally be the most common.  I'd also like to examine automatic discovery of library translations moving forward as well.